### PR TITLE
Homepage update

### PR DESCRIPTION
**Pull request by mejackreed**
This PR does the following things:

 - Adds byebug for debugging
 - Updates the homepage for improved visual design
   - Features the search bar more prominently with a new call to action
   - Removes all facet links in favor of important featured searches
   - Adds more context to map, the aim here is to help a non-technical user

screen shot 2018-01-24 at 9 25 58 am

### CODE REVIEW

File: app/models/concerns/featured_content_behavior.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    end
+
+    solr_params
+  rescue NoMethodError
+    solr_params
+  end
+
+  private
+
+  def scanned_maps_content_params
```
Comment (drh-stanford) [inline app/models/concerns/featured_content_behavior.rb]: what do the Blacklight breadcrumbs look like for these queries?

File: config/locales/blacklight.en.yml [hunk 2]
```diff
@@ -5,10 +5,10 @@ en:
       form:
         search:
           label: 'search for'
-          placeholder: ''
+          placeholder: 'Try "groundwater"'
```
Comment (drh-stanford) [inline config/locales/blacklight.en.yml]: I find this helper tip a little confusing -- are there other suggestions?

File: app/models/concerns/featured_content_behavior.rb (hunk 1 above)
Comment (mejackreed) [reply app/models/concerns/featured_content_behavior.rb]: screen shot 2018-01-24 at 11 47 18 am

File: config/locales/blacklight.en.yml (hunk 2 above)
Comment (mejackreed) [reply config/locales/blacklight.en.yml]: perhaps just "groundwater" or leave it blank entirely?

Comment (drh-stanford) [reply config/locales/blacklight.en.yml]: how about "Search for data and maps"? or blank is ok with me. The former begs the question whether we need that header. This isn't my forte though :).

Comment (mejackreed) [reply config/locales/blacklight.en.yml]: Yep I just stole this from Airbnb:

screen shot 2018-01-24 at 12 25 05 pm

I'm inclined to leave it as is.. and see if we get any feedback. It's a quick change if we have to modify it.

**Pull request merged**